src/guides/whats-new.md

```diff
@@ -17,6 +17,63 @@ and follow the [Dart blog][].
 [dart-announce]: https://groups.google.com/a/dartlang.org/d/forum/announce
 [Dart blog]: https://medium.com/dartlang
 
+## November 15, 2023: 3.2 release
+
+This section lists notable changes made from August 17, 2023,
+through November 15, 2023.
+For details about the 3.2 release,
+check out [3.2 blog post][] and the and the [SDK changelog][3-2-changelog].
+
+[3.2 blog post]: /
+[3-2-changelog]: https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md#320
+
+### Docs updated or added to dart.dev
+{: .no_toc}
+
+In addition to bug fixes and incremental improvements,
+we made the following changes to this site:
+
+* Updated the [Fixing type promotion failures][no-promo] page
+  for a new type promotion feature, private final field promotion.
+  * Made minor related adjustments to the Effective Dart entry on
+    [type promotion][], the [Understanding Null Safety][] page, and other various
+    places across the site.
+* Introduced documentation on the experimental native assets feature
```

Comment:
Consider linking directly to https://dart.dev/guides/libraries/c-interop#native-assets